drivers/adc/adc_lmp90xxx.c

```diff
@@ -595,7 +595,7 @@ static int lmp90xxx_adc_read_channel(const struct device *dev,
 	}
 
 	if (LMP90XXX_HAS_DRDYB(config)) {
-		k_sem_take(&data->drdyb_sem, K_FOREVER);
+		k_sem_take(&data->drdyb_sem, ADC_CONTEXT_WAIT_FOR_COMPLETION_TIMEOUT);
```

Comment:
It is still the default `K_FOREVER` here. What was the intention of that change?
For `ADC_CONTEXT_WAIT_FOR_COMPLETION_TIMEOUT` defined to something else, the return value should not be just ignored.

Reply:
done